flank-actions/sendMessage.kts

```diff
@@ -0,0 +1,15 @@
+import kotlin.system.exitProcess
+
+
+//DEPS com.github.kittinunf.fuel:fuel:2.2.3
+
+//INCLUDE slackService.kt
+//INCLUDE common.kt
+
+val result = sendMessage(args)
+
+if (result != 0){
+    exitProcess(result)
+}
+
+println("Message has been sent with result $result")
```

Comment:
How about?

```suggestion
if (sendMessage(args) != 0){
    exitProcess(result)
}

println("Message has been sent")
```

Because there will be always `0` as result

Reply:
Ive changed it as this could be modified in the future so I have moved the printLn elsewhere.